okhttp/src/jvmMain/kotlin/okhttp3/Request.kt

```diff
@@ -82,6 +84,11 @@ actual class Request internal actual constructor(builder: Builder) {
 
   actual fun headers(name: String): List<String> = commonHeaders(name)
 
+  @JvmName("reifiedTag")
```

Comment:
Is this intended to be called, or just moved out of the way?

Reply:
Yep, this is for calling. It needs a name to avoid colliding with the parameterless signature.

Reply:
Should you test for this as new public Java API?

BTW not seeing it in the jvmApiCheck

```
> API check failed for project okhttp.
  --- /home/runner/work/okhttp/okhttp/okhttp/api/okhttp.api
  +++ /home/runner/work/okhttp/okhttp/okhttp/build/api/okhttp.api
  @@ -1011,6 +1011,7 @@
   	public final fun newBuilder ()Lokhttp3/Request$Builder;
   	public final fun tag ()Ljava/lang/Object;
   	public final fun tag (Ljava/lang/Class;)Ljava/lang/Object;
  +	public final fun tag (Lkotlin/reflect/KClass;)Ljava/lang/Object;
   	public fun toString ()Ljava/lang/String;
   	public final fun url ()Lokhttp3/HttpUrl;
   }
  @@ -1034,6 +1035,7 @@
   	public fun removeHeader (Ljava/lang/String;)Lokhttp3/Request$Builder;
   	public fun tag (Ljava/lang/Class;Ljava/lang/Object;)Lokhttp3/Request$Builder;
   	public fun tag (Ljava/lang/Object;)Lokhttp3/Request$Builder;
  +	public final fun tag (Lkotlin/reflect/KClass;Ljava/lang/Object;)Lokhttp3/Request$Builder;
   	public fun url (Ljava/lang/String;)Lokhttp3/Request$Builder;
   	public fun url (Ljava/net/URL;)Lokhttp3/Request$Builder;
   	public fun url (Lokhttp3/HttpUrl;)Lokhttp3/Request$Builder;
  
   You can run :okhttp:apiDump task to overwrite API declarations
```